jenkins/windows_java.cmd

```diff
@@ -63,3 +63,9 @@ cd %build-root%
 call mvn install -DskipITs=false
 if errorlevel 1 goto :eof
 cd %build-root%
+
+REM -- Android Test Build --
```

Comment:
Can you create a new script only for Android ? copy only what you need from windows_java.cmd to android_e2e.cmd (possible you would only need mvn install) and then following that trigger Android tests build.